debug: true
bypass_behavior: false
planner_setup:
  via_points_speed_linear_ratio: 1.5
robot:
  wheelbase: 2.7
  min_turning_radius: 10.4
  max_vel_x: 5.5
  max_vel_y: 0.0
  acc_lim_theta: 0.23
  acc_lim_x: 2.4
  max_vel_x_backwards: 0.01
  cmd_angle_instead_rotvel: true
teb_setup:
  homotopy_enabled: false
teb_trajectory:
  dt_ref: 0.5
  dt_hysteresis: 0.05
  min_obstacle_distance: 2.3
  inflation_distance: 0.2
  xy_goal_tolerance: 0.2
  yaw_goal_tolerance: 0.2
teb_weights:
  no_inner_iterations: 4
  no_outer_iterations: 5
  penalty_epsilon: 0.05
  weight_obstacle: 50.0
  weight_inflation: 0.1
  weight_viapoints: 1.0
  weight_optimaltime: 1.0
  weight_shortest_path: 0.0
  weight_acc_lim_x: 1.0
  weight_acc_lim_y: 0.0
  weight_acc_lim_theta: 1.0
  weight_max_vel_x: 1.0
  weight_max_vel_y: 0.0
  weight_max_vel_theta: 0.0
  weight_kinematics_turning_radius: 1.0
  weight_kinematics_forward_drive: 0.001
  weight_kinematics_nh: 1000.0
  weight_adapt_factor: 2.0
  obstacle_cost_exponent: 1.0
